### PR TITLE
Configure parametric test to get the cpp tracer in same way as system-tests

### PR DESCRIPTION
**Pull request by robertomonteromiguel**
Configure parametric test to get the cpp tracer in same way as system-tests

Related with system-tests PR: https://github.com/DataDog/system-tests/pull/1983

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -105,12 +105,15 @@ jobs:
...
+
       - run:
-          name: Run
+          name: Run Parametric tests
           environment:
             TEST_LIBRARY: cpp
             DD_TRACE_CPP_COMMIT: << pipeline.git.revision >>
-          command: cd system-tests && ./build.sh -i runner && ./run.sh PARAMETRIC --log-cli-level=DEBUG 
+          command: |
+            echo "https://github.com/DataDog/dd-trace-cpp@$DD_TRACE_CPP_COMMIT" > ~/project/system-tests/binaries/cpp-load-from-git
```
Comment (dmehala) [inline .circleci/config.yml:115]: Does this actually work?

Comment (robertomonteromiguel) [reply .circleci/config.yml:115]: If it will work once we merge the PR: https://github.com/DataDog/system-tests/pull/1983

**Review by dmehala — APPROVED** [review]
LGTM :shipit:

File: .circleci/config.yml (hunk 1 above)
Comment (dmehala) [reply .circleci/config.yml:115]: Could you just set the version of `dd-trace-cpp` instead of the whole github url?

Comment (robertomonteromiguel) [reply .circleci/config.yml:115]: It is to maintain a certain similarity with other tracers. ie: https://github.com/DataDog/system-tests/blob/main/utils/scripts/load-binary.sh#L230

**Pull request merged**